src/lib/pickles/impls.ml

```diff
@@ -107,9 +118,14 @@ module Wrap = struct
     type t = Field.t
 
     let forbidden_shifted_values =
-      forbidden_shifted_values ~size_in_bits:Constant.size_in_bits
-        ~modulus:(Step.Impl.Bigint.to_bignum_bigint Constant.size) ~f:(fun x ->
-          Impl.Bigint.(to_field (of_bignum_bigint x)))
+      let f x = Impl.Bigint.(to_field (of_bignum_bigint x)) in
+      let modulus = Step.Impl.Bigint.to_bignum_bigint Constant.size in
+      let size_in_bits = Constant.size_in_bits in
+      let values = forbidden_shifted_values ~size_in_bits ~modulus in
+      let larger_than_mod x =
```

Comment:
For consistency with the changes you'll have to make above, can you move this into `f`?